vendor/arbiter/arbiter.cpp

```diff
@@ -2886,7 +2886,10 @@ std::string AZ::Config::extractBaseUrl(
     return account + "." + service + "." + endpoint + "/";
 }
 
-std::unique_ptr<std::size_t> AZ::tryGetSize(std::string rawPath) const
+std::unique_ptr<std::size_t> AZ::tryGetSize(
+    const std::string rawPath,
```

Comment:
Is there some reason for `const` in these arguments without passing by reference?

Reply:
I don't know. It's the signature of the function in `Http` class. It's how @connormanning wrote it I guess.

Reply:
It's just weird to need a copy that you're not going to modify, but it's not a big deal.

Reply:
Maybe some refactoring ahead in `arbiter`.

Reply:
I just like making things `const` by default when possible so I've adopted the habit of const-ifying in this context.  Note that this is in the cpp implementation, the header interface does not have this `const` so if for some reason this parameter copy had to be changed to be mutable in the implementation, the caller wouldn't notice.